File: Frontend/Angular/src/app/pages/dottuyensinh/model/dot-tuyen-sinh.model.ts
import { BaseModel } from "src/app/_metronic/core/models/_base.model";
export class DotTuyenSinhModel extends BaseModel {
    Id: number;
    NamHoc?: number | null;
    Dot?: number | null;
    TenDotTS?: string | null;
    KhoaHoc?: number | null;
    ThoiGianNhanHSTuNgay?: Date | string | null;
    ThoiGianNhanHSDenNgay?: Date | string | null;
    NgayInGBTT?: Date | string | null;
    ThoiGianLayHSTuNgay?: Date | string | null;
    ThoiGianLayHSDenNgay?: Date | string | null;
    NgayNhapHocDK?: Date | string | null;
    GhiChu?: string | null;
    CreatedBy?: string | null;
    CreatedDate?: Date | string | null;
    HienThi?: boolean;
    KichHoat?: boolean;
    TenKhoaHoc?: string;
    TenNamHoc?: string;

    clear() {
        this.Id = 0;
        this.NamHoc = 0;
        this.Dot = 0;
        this.TenDotTS = '';
        this.KhoaHoc = 0;
        this.ThoiGianNhanHSTuNgay = null;
        this.ThoiGianNhanHSDenNgay = null;
        this.NgayInGBTT = null;
        this.ThoiGianLayHSTuNgay = null;
        this.ThoiGianLayHSDenNgay = null;
        this.NgayNhapHocDK = null;
        this.GhiChu = '';
        this.CreatedBy = '';
        this.CreatedDate = null;
        this.HienThi = true;
        this.KichHoat = true;
        this.TenKhoaHoc = '';
        this.TenNamHoc = '';
    }

}

File: SampleCodeAPI/Model/DotTuyenSinhModel.cs
namespace SampleCodeAPI.Model
{
    public class DotTuyenSinhModel
    {
        public int? Id { get; set; }
        public int NamHoc { get; set; } 
        public int Dot { get; set; }    
        public string TenDotTS { get; set; }
        public int KhoaHoc { get; set; }
        public DateTime? ThoiGianNhanHSTuNgay { get; set; }
        public DateTime? ThoiGianNhanHSDenNgay { get; set; }
        public DateTime? NgayInGBTT { get; set; } 
        public DateTime? ThoiGianLayHSTuNgay { get; set; }
        public DateTime? ThoiGianLayHSDenNgay { get; set; }
        public DateTime? NgayNhapHocDK { get; set; } 
        public string? GhiChu { get; set; }
        public bool? HienThi { get; set; }
        public bool? KichHoat { get; set; }
        public int? DeletedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? DeletedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string? TenKhoaHoc { get; set; }
        public string? TenNamHoc { get; set; }
    }
}
